src/modules/drawing/lib/ProfileChart.js

```diff
@@ -279,9 +279,12 @@ export default class ProfileChart {
 
     updateLabels() {
         this.group
-            .select('text.profile-label-x')
+            .select('text.ga-profile-label-x')
+            .attr('x', this.width / 2)
             .text(`${i18n.global.t(this.options.xLabel)} [${this.unitX}]`)
-        this.group.select('text.profile-label-y').text(`${i18n.global.t(this.options.yLabel)} [m]`)
+        this.group
+            .select('text.ga-profile-label-y')
+            .text(`${i18n.global.t(this.options.yLabel)} [m]`)
```

Comment:
Could you replace the class `ga-profile-label` with its version without `ga-`? That was the prefix used in `mf-geoadmin3` and I'd like to remove it so that we can differentiate part that are straight copy paste and those that we have already processed

Reply:
I fixed that